packages/core/tools/_links.scss

```diff
@@ -83,6 +86,8 @@
   &:focus {
     color: $nhsuk-link-focus-color;
     text-decoration: underline;
+    outline: $nhsuk-focus-width solid transparent;
```

Comment:
@davidhunter08 - Outline stuff needs to come before the text-decoration property as it's causing a linting issue (alphabetical order)

Reply:
Have fixed.